classes/core.php

```diff
@@ -3950,25 +3950,39 @@ static public function render_field($field, $form = null, $entry_data = array(),
 
 		$type = Caldera_Forms_Field_Util::get_type($field, $form);
 
+		// Logic for grouped controls
+		if ($type == "radio" || $type == "checkbox") {
+			$wrapper_before = "<fieldset data-field-wrapper=\"" . $field['ID'] . "\" class=\"" . $field_wrapper_class . "\" id=\"" . $field_id_attr . "-wrap\">\r\n";
```

Comment:
Just a suggestion: I think it would be better to use `sprintf` instead of endless concatenating vars. Something like `$wrapper_before = sprintf('<fieldset data-field-wrapper="" ...>', $field['ID'], ...)`.

It would greatly improve readability ;)

Reply:
Alternatively, you could just make use of PHP's inline variables. Just make sure you wrap the whole thing into double quotes and vars in curly braces:

`$wrapper_before = "<fieldset data-field='{$field['ID']}'> ... ";`